src/components/app/App.js

```diff
@@ -2,21 +2,45 @@ import React, { Component } from 'react'
 import './App.css'
 
 import movieData from '../../data/movieData'
-import Movies from '../Movies/Movies'
+import Movies from '/Users/tramtram/turing_work/3mod/projects/2week/rancid-tomatillos/src/components/movies/Movies.js'
 import MoviePage from '../moviepage/MoviePage'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData,
-      selectedMovie: ''
+      allMovies: 'movieData',
+      selectedMovie: '',
+      allMoviesLoading: true,
+      singleMovieLoading: true,
+      error: ''
     }
   }
 
+  componentDidMount = () => {
+    // console.log('Component Did Mount')
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => response.json())
+      .then(data => this.setState({allMovies: data, allMoviesLoading: false}))
+      // .then(() => console.log('static data', this.state.allMovies))
+      .catch(err => this.setState({error: err}))
+  }
+
+  getSingleMovie = (id) => {
+    const url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/' + id
+    fetch(url)
```

Comment:
Same as above - be sure to move the fetch into it's own file. But I like your use of the URL arg here - it means you could potentially have 1 fetch for both all movies and single movie, nice!

Reply:
we could still use concatenate after we move the fetch call to a separate file right?